specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json

```diff
@@ -2177,6 +2306,30 @@
         }
       }
     },
+    "InboundEnvironmentEndpoint": {
+      "description": "Endpoints for a particular type",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "Text describing the endpoints.",
+          "type": "string"
+        },
+        "endpoints": {
+          "description": "The endpoint ip addresses in cidr notation.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "ports": {
```

Comment:
Given that endpoints and ports are separate arrays, how does this work in practice? Each endpoint IP address supports each port?